src/WireMock.Net/Matchers/JSONPathMatcher.cs

```diff
@@ -67,7 +67,7 @@ public MatchResult IsMatch(string? input)
         var score = MatchScores.Mismatch;
         Exception? exception = null;
 
-        if (input != null)
+        if (!string.IsNullOrWhiteSpace(input))

```

Comment:
Can you also add a test for empty string based on this test?
https://github.com/WireMock-Net/WireMock.Net/blob/master/test/WireMock.Net.Tests/Matchers/JsonPathMatcherTests.cs#L55